### PR TITLE
Use kotlin-collections import alias to not uselessly self-call in Set…

### PR DESCRIPTION
**Pull request by glamm**
…K.combineAll

### CODE REVIEW

File: modules/core/arrow-core/src/test/kotlin/arrow/core/extensions/setkTest.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import arrow.core.SetK
+import arrow.core.extensions.setk.monoid.monoid
+import arrow.core.k
+import arrow.test.UnitSpec
+import io.kotlintest.shouldBe
+
+class setkTests: UnitSpec() {
+  init {
+    "SetK combineAll leads to StackOverflow #1717" {
```
Comment (pakoito) [inline modules/core/arrow-core/src/test/kotlin/arrow/core/extensions/setkTest.kt]: Love the error-linked test

Comment (pakoito) [reply modules/core/arrow-core/src/test/kotlin/arrow/core/extensions/setkTest.kt]: Could you please move it with the other tests in SetKTests in core?

**Comment by pakoito** [comment]
In the end @devesh-shetty was already working on a PR that added missing tests such as this one, and it caught the error even if he didn't know about these. It's merged already: https://github.com/arrow-kt/arrow/pull/1705, so should we close this now?

**Comment by pakoito** [comment]
Bob said it's okay to merge!

**Comment by devesh-shetty** [comment]
> Bob said it's okay to merge!

But isn't the fix already in? 🤔

**Comment by pakoito** [comment]
*close

**Pull request closed without merging**